datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -2481,6 +2492,34 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn bitwise_shift_array_test() -> Result<()> {
+        let input = Arc::new(Int32Array::from(vec![Some(2), None, Some(10)])) as ArrayRef;
+        let modules =
```

Comment:
I suggest a test for when the `modules` is Null (you cover NULL for the input already)